frontend/public/config.yml

```
@@ -290,12 +245,7 @@ collections:
     fields:
       - { label: 'Názov', name: 'title', widget: 'string' }
       - { label: 'Text', name: 'body', widget: 'markdown' }
-      - {
-          label: 'LAyout',
-          name: 'layout',
-          widget: 'hidden',
-          default: '../layouts/DocumentLayout.astro',
-        }
+      - { label: 'LAyout', name: 'layout', widget: 'hidden', default: '../layouts/DocumentLayout.astro' }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)